CC.UI.Webhost/Infrastructure/HtmlExtensionHelpers.cs

```diff
@@ -29,6 +29,24 @@ public static class HtmlExtensionHelpers
             return MvcHtmlString.Create(imgHtml);
         }
 
+        public static MvcHtmlString ImageTag(this HtmlHelper html, WebImageOCC src, string alt, int? height = null, int? width = null,
```

Comment:
I know we dont have many tests, but it would be nice to change that. This method seems a good candidate for a unit test.